src/marqo/core/inference/embedding_models/hugging_face_model.py

```diff
@@ -119,8 +146,12 @@ def _load_from_zip_file(self) -> Tuple:
 
         model_dir = self.extract_huggingface_archive(zip_file_path)
         try:
-            model = AutoModel.from_pretrained(model_dir).to(self.device)
-            tokenizer = AutoTokenizer.from_pretrained(model_dir)
+            model = AutoModel.from_pretrained(model_dir).to(
```

Comment:
typo here? **kwargs should be in side from_pretrained?

Reply:
Addressed